src/lib/utils/meta_tables/meta_chunk_sort_orders_table.cpp

```diff
@@ -30,9 +30,9 @@ std::shared_ptr<Table> MetaChunkSortOrdersTable::_on_generate() const {
       const auto ordered_by = chunk->ordered_by();
       if (ordered_by) {
         std::stringstream order_by_mode_steam;
-        order_by_mode_steam << ordered_by->second;
+        order_by_mode_steam << ordered_by->front().second;
```

Comment:
The reason why this table exists and the sort column is not part of `meta_chunks` is that we can have multiple order columns. So, `front()` should be used here, please add all columns (which is currently always the same, but let's do it right from the start).

Reply:
Could you paraphrase what you mean here, please?

Reply:
When there would be only one sort column per chunk, we could store that column name in the `meta_chunks` table. The reason for the `meta_sort_orders` table is that chunks might be sorted by multiple columns.
As such, please do not simply put `front()` into the table, but all elements of the sorted vector per chunk.

Reply:
Okay, we changed it to outputting a row for each `ordered_by` entry.